README.md

```diff
@@ -236,7 +236,93 @@ TODO: Add notes about how to use the system.
 
 ## 🚀 Deployment <a name = "deployment"></a>
 
-TODO: Add additional notes about how to deploy this on a live system.
+### Prerequisites
+
+What things do you need to install the software, and how to install them.
+
+[git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+[docker compose](https://docs.docker.com/compose/install/)
+
+[make](https://www.gnu.org/software/make/)
+
+> **_NOTE:_**  make is optional, but you may find that having it greatly simplifies your life.
+
+### Installing
+
+Register a new OAuth application on the [42 intranet](https://profile.intra.42.fr/oauth/applications/new)
+
+Set the Redirect URIs
+
+```
+http://your-domain-or-ip-here/api/v1/auth/login
+http://localhost/api/v1/auth/login
+```
+
+Clone the GitHub repository
+
+```shell
+git clone https://github.com/42AGV/ft_transcendence.git
+```
+
+
+Change the current working directory to `ft_transcendence`
+
+```shell
+cd ft_transcendence
+```
+
+Either create a `.env` file in the root directory containing the following environment variables (replace the FORTYTWO_APP_ID and FORTYTWO_APP_SECRET with the ones from the OAuth application you create in the previous step):
```

Comment:
If we are going to use `.env` from now, shoudn't be `.env.sample` and `.env.test` deleted?